tests/sentry/eventstream/kafka/test_consumer.py

```diff
@@ -156,4 +158,4 @@ def test_post_process_forwarder_streaming_consumer(self, dispatch_post_process_g
             occurrence_id=None,
         )
 
-        consumer.signal_shutdown()
+        consumer._shutdown()
```

Comment:
I wasn't able to address the test failure here w/o changing this - it always hung and gave an error about the `sentry_test` database already existing. It seems like `signal_shutdown` is meant for `run` but we used `run_once` so `_shutdown` is appropriate.